sdks/python/apache_beam/runners/interactive/cache_manager.py

```diff
@@ -202,7 +202,8 @@ class SafeFastPrimitivesCoder(coders.Coder):
   # pylint: disable=deprecated-urllib-function
 
   def encode(self, value):
-    return quote(coders.coders.FastPrimitivesCoder().encode(value))
+    return quote(coders.coders.FastPrimitivesCoder().encode(value)).encode(
+        'utf-8')
 
   def decode(self, value):
     return coders.coders.FastPrimitivesCoder().decode(unquote_to_bytes(value))
```

Comment:
Do we need to also `s/value/value.decode('utf-8')/` in Line 209 in this case?

Reply:
No, `unquote_to_bytes` unquotes from `bytes` to `bytes` here.
The problem was that `quote` always returns an ASCII string, so it needs to be encoded again.